sympy/core/tests/test_power.py

```diff
@@ -526,3 +526,11 @@ def test_issue_14815():
 def test_issue_18509():
     assert unchanged(Mul, oo, 1/pi**oo)
     assert (1/pi**oo).is_extended_positive == False
+
+
+def test_issue_18762():
+    from sympy.abc import psi, epsilon
```

Comment:
I think that `symbols` should be used here, and `e` instead of `epsilon` for brevity.